### ISSUE TITLE
Need a function to clean up variable names with whitespace...

### ISSUE THREAD

**Issue by rosseji** [rosseji/dev#11]
In item level data frame the variable names can be an issue when they have whitespace...

ie 
```
df_IL %>% 
   select(Final Total)
> error!
```
![](http://thumbpress.com/wp-content/uploads/2013/05/Funniest-Fail-GIFs-2.gif)

`Final Total` to `Final.Total`

```
colnames()
# or
purrr::set_names()
```

**Comment by aidandmorrison** [comment]
Sounds like a good idea... I would think that it should be easy enough using Stringr to do something like that!  Is that a suggested task for me in spare time? ;-)

**Comment by rosseji** [comment]
Yep `stringr` will be great! I think the exact function is `str_replace_all`

**Comment by aidandmorrison** [comment]
cool.  Will ahve a shot on weekend
